examples/java/src/main/java/org/apache/beam/examples/WordCount.java

```diff
@@ -196,6 +197,27 @@ public String create(PipelineOptions options) {
       }
     }
 
+    /**
+     * Return default input file path according to runner type.
+     *
+     * <p><ul>
+     *   <li>SparkRunner:
+     *   .../src/test/resources/LICENSE</li>
+     *   <li>other runners:
+     *   gs://dataflow-samples/apache/LICENSE</li>
+     * </ul>
+     */
+    public static class InputFactory implements DefaultValueFactory<String> {
```

Comment:
This class is used for setting a default test data path if people don't specify `--inputFile` in options. Also people can use `--inputFile` to customize their own test data. Can this answer your question?